core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -990,7 +989,12 @@ portfolio.failed.cantUnfail=This trade cannot be moved back to open trades at th
   Try again after completion of trade(s) {0}
 portfolio.failed.depositTxNull=The trade cannot be reverted to a open trade. Deposit transaction is null.
 portfolio.failed.delayedPayoutTxNull=The trade cannot be reverted to a open trade. Delayed payout transaction is null.
+portfolio.failed.penalty.msg=This will charge the buyer as maker the trade fee of {0} XMR and return the remaining trade funds of {1} XMR to their wallet. Are you sure you want to send?\n\n\
```

Comment:
For a failed trade, the arbitrator can charge either the maker or taker, who might be the buyer or seller.